website_text/editorial_board.md

```diff
@@ -1,16 +1,17 @@
 ## Editors
 
-**Managing Editors**: David L. Mobley, Michael Shirts, Daniel M. Zuckerman
+**Managing Editors**: David L. Mobley, Michael R. Shirts, Daniel M. Zuckerman
 
-**Lead Editor for Best Practice Guides**: Michael Shirts
-
-**Lead Editor for Lessons Learned**: Daniel M. Zuckerman 
+**Lead Editor for Tutorials**: Rommie Amaro
```

Comment:
The comments say you're now being consistent about middle initial, but there is no middle initial for Amaro, Piquemal, Siderius, or several others.

Reply:
Consistent at this point just means that names just appear in one way, with or without an initial.  Should probably survey to ask what people want.

Reply:
Can we just push this through since it's better than it currently is, and create an action item to get everyone's preferred name?

Reply:
Yes, just need conflicts resolved. Maybe I can do that real quick.